app/views/setup/welcome.html.slim

```diff
@@ -20,23 +21,43 @@ h1 Initial CaaSP Configuration
               i class='glyphicon glyphicon-info-sign'
 
   .panel.panel-default
-    .panel-heading Container engine proxy settings
-    .panel-body
-      p Proxy server the container engine will use to download images.
-      p Leave them empty if no proxy server is in place inside of your organization.
-      .form-group
-        = f.label :http_proxy, "HTTP proxy"
-        = f.url_field :http_proxy, value: Pillar.value(pillar: :http_proxy), class: "form-control"
-      .form-group
-        = f.label :https_proxy, "HTTPS proxy"
-        = f.url_field :https_proxy, value: Pillar.value(pillar: :https_proxy), class: "form-control"
+    .panel-heading.clearfix
+      | Proxy settings
       .form-group
```

Comment:
You can remove `.form-group` (it will remove the margin-bottom) since the button group is not being used in a typical form.